src/shared/components/ShortcutIndicator/ShortcutIndicator.styles.tsx

```diff
@@ -0,0 +1,19 @@
+import styled from '@emotion/styled'
+
+import { colors, sizes, square, typography } from '@/shared/theme'
+
+export const ShortcutIndicator = styled.kbd`
+  ${square(16)};
+
+  font-family: ${typography.fonts.base};
+  color: ${colors.gray[300]};
+  font-weight: ${typography.weights.extraBold};
+  font-size: 0.7rem;
```

Comment:
Since here we set the explicit size for the container, maybe let's use an explicit (in `px`) font size as well?